docs/src/reference/nonlinear.md

```diff
@@ -31,9 +31,13 @@ features_available
 eval_objective
 eval_constraint
 eval_objective_gradient
+hessian_objective_structure
+eval_hessian_objective
 jacobian_structure
 hessian_lagrangian_structure
 eval_constraint_jacobian
+hessian_constraint_structure
+eval_hessian_constraint
```

Comment:
Ditto with `eval_constraint_hessian`

Reply:
Yes, I agree with you  but I wanted to be consistent with `eval_hessian_lagrangian` that is here since MPB.
Maybe we should add `eval_lagrangian_hessian` and deprecated `eval_hessian_lagrangian`?

Reply:
Ah. Perhaps this is okay then. I'll re-order the docs to group things together. I don't really want to start deprecating existing methods.